firebase.json

```diff
@@ -22,7 +22,7 @@
                     },
                     {
                         "key": "Content-Security-Policy",
-                        "value": "child-src 'self' blob:;default-src 'self'; script-src 'self' 'unsafe-inline' https://www.google-analytics.com  https://apis.google.com blob:; connect-src 'self' https://stats.g.doubleclick.net https://*.tiles.mapbox.com https://api.mapbox.com https://events.mapbox.com https://*.entur.io https://*.entur.org https://*.cloudfunctions.net https://*.googleapis.com https://www.google-analytics.com; style-src 'self' 'unsafe-inline'; img-src 'self' blob: data:  https://www.google.no  https://www.google.com https://*.googleapis.com https://www.google-analytics.com; object-src 'none'; frame-ancestors https:; manifest-src 'self' blob:"
+                        "value": "child-src 'self' blob:;default-src 'self'; script-src 'self' 'unsafe-inline' https://www.google-analytics.com  https://apis.google.com blob:; connect-src 'self' https://stats.g.doubleclick.net https://*.tiles.mapbox.com https://api.mapbox.com https://events.mapbox.com https://*.entur.io https://*.entur.org https://*.cloudfunctions.net https://*.googleapis.com https://www.google-analytics.com; style-src 'self' 'unsafe-inline'; img-src 'self' blob: data:  https://www.google.no  https://www.google.com https://*.googleapis.com https://www.google-analytics.com; object-src 'none'; frame-ancestors https:; manifest-src 'self' blob:; frame-src https://entur-tavla-staging.firebaseapp.com/ https://entur-tavla-prod.firebaseapp.com/"
```

Comment:
Sånn eg skjønner det så er `frame-src` det som tillates å iframe inn på Tavla. Men tavla har ingen iframes hos seg, så det burde ikkje vere relevant. 🤔

Reply:
Ta med `'self'` kanskje for sikkerhets skyld?

Reply:
![image](https://user-images.githubusercontent.com/32391231/133571029-50cd0df1-5195-48a0-93e9-569fbfa0a83f.png)

Den får en injisert av det scriptet som hentes fra apis.google.com. Ref meldingene på slack skjønner jeg ikke hva hensikten med den er. Dette [her](https://stackoverflow.com/questions/68692470/firebase-v9-loads-a-large-iframe-js-263k-on-mobile) er det nærmeste jeg har funnet svar, men ble ikke så veldig klok på det.

Reply:
Og det virker ikke som man har så mye valg om å la være å godta den om man vil at authen skal fungere 😕